app/services/clients/connector.client.js

```diff
@@ -397,6 +397,21 @@ ConnectorClient.prototype = {
     return response.data
   },
 
+  /**
+   * @param {string} serviceId
+   * @param {string} accountType
+   * @param {string} emailText
+   */
+  updateConfirmationEmailByServiceId: async function (serviceId, accountType, emailText) {
```

Comment:
should this be `updateConfirmationTemplateByServiceIdAndAccountType`?

Reply:
I've changed this to `patchEmailNotificationByServiceIdAndAccountType: async function (serviceId, accountType, payload)`, with the goal of making all callers that want to patch email notifications by service id and account type use this method. This will be done in the next PR.